spec/mailers/admin_mailer_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe AdminMailer, type: :mailer do
+  describe 'send notification email' do
+    let!(:admin) { create(:user, :admin, organization: organization) }
+    let!(:organization) { create(:organization) }
+    let!(:channel) { create(:channel, organization: organization) }
+    let!(:mail) { AdminMailer.delete_channel_email(channel) }
+
+    it 'deliver success' do
+      expect { channel.destroy }.to change { ActionMailer::Base.deliveries.count }.by(1)
```

Comment:
no context, pls make test structure more explicit to understand what and why code is doing